localstorage/localstorage-todo.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Todo</title>
+    </head>
+
+    <body>
+
+        <form id="addPlaces">
+            <input type="text" placeholder="Add Place" />
+            <input type="submit" />
+        </form>
+        <ul id="listWrapper">
+        </ul>
+
+        <script>
+            let addPlaces = document.forms["addPlaces"];
+            let list = document.getElementById("listWrapper");
+
+            let placeArray = localStorage.getItem('place') ? localStorage.getItem('place').split(",") : [];
+
+            const createTodo = (inpVal) => {
+                let li = document.createElement("li");
```

Comment:
Convert the entire structure to Template String. Generate the html from it and then attach listeners to it.